Doc/distutils/apiref.rst

```diff
@@ -1452,6 +1452,8 @@ name.
 
 .. module:: distutils.sysconfig
    :synopsis: Low-level access to configuration information of the Python interpreter.
+.. deprecated:: 3.10
+   :mod:`distutils.sysconfig` has been merged to :mod:`sysconfig`.
```

Comment:
merged with or replaced by?

Reply:
I'm not sure. "replaced by" sounds to me like the `distutils.sysconfig` is no longer available which is not true. The key message here is that the module is deprecated and you can find all its functionalities on the `sysconfig`.
But I can definitely change it if you think that "replaced by" is better.

Reply:
Ah from the most recent discussion after the PEP was accepted I thought distutils would be fully unavailable in the standard library, but now I remember this PR is only about sysconfig.